lib/commands/gesture.js

```diff
@@ -217,6 +238,61 @@ helpers.handleLongPress = async function (gestures) {
   return await this.proxyCommand(endpoint, 'POST', params);
 };
 
+helpers.handlePinchOrZoom = async function (actions) {
+  // currently we can only do this action on an element
+  if (!actions[0][0].options.element ||
+      actions[0][0].options.element !== actions[1][0].options.element) {
+    log.errorAndThrow('Pinch/zoom actions must be done on a single element');
+  }
+  let el = actions[0][0].options.element;
+
+  function determineScale (x, y, pinch) {
```

Comment:
even so, it seems meaty enough that it could stand on its own. maybe it's a personal nitpick, I just try to have as little depth as possible when defining functions